### PR TITLE
Update suggested constraints for build_runner

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @natebosch

**Comment by kevmoo** [comment]
@chalin – uh – I think I did something wrong...

**Comment by kwalrath** [comment]
We need to update the example file.

**Comment by kwalrath** [comment]
I can do it.

**Comment by chalin** [comment]
Btw, I usually update all example app pubspecs at the same time. I can do that tomorrow, and ensure that code excerpts are refreshed too. Shall I commit to this PR or do you want me to handle that separately?

**Comment by kwalrath** [comment]
They did not go green. :( Let's let Patrice fix this tomorrow.

**Comment by kevmoo** [comment]
SGTM

**Comment by kevmoo** [comment]
Letting @chalin run w/ this...

**Comment by chalin** [comment]
See #1470

File: examples/ng/doc/quickstart/pubspec.yaml [hunk 1]
```diff
@@ -14,9 +14,10 @@ dev_dependencies:
   # #enddocregion build-dependencies
   angular_test: ^2.0.0-alpha
   # #docregion build-dependencies
-  build_runner: ^0.8.0
+  build_runner: ^0.8.2
+  build_web_compilers: ^0.3.6
+  # If you write tests that verify built output
```
Comment (chalin) [inline examples/ng/doc/quickstart/pubspec.yaml]: @kevmoo - FYI, I didn't add this comment to #1470.

TL;DR
I believe that this comment should read something like: you'll need `build_test` if you test your web app. I usually favor having such comments in the prose, and there is one already in the [build_runner](https://webdev-dartlang-org-dev.firebaseapp.com/tools/build_runner) page:

> The `build_test` package is optional; add it if you'll be testing your app.

So I didn't repeat that comment in the `pubspec.yaml` file(s) of #1470.

Comment (kevmoo) [reply examples/ng/doc/quickstart/pubspec.yaml]: Ack
